packages/core/RNAnalytics.podspec

```diff
@@ -16,11 +16,11 @@ Pod::Spec.new do |s|
   s.author              = { 'Segment' => 'friends@segment.com' }
   s.source              = { :git => 'https://github.com/segmentio/analytics-react-native.git', :tag => s.version.to_s }
 
-  s.platform            = :ios, '10.0'
+  s.platform            = :ios, '11.0'
```

Comment:
@bsneed Is this required? It's a breaking change.

https://github.com/facebook/react-native#-requirements still supports `10.0`

Reply:
Yes, it is required.  Our iOS lib minimum is now 11.0.  That's my bad on not bumping the version #'s appropriately given it was indeed a breaking change.  My apologies for that.  If you need 10.x support, we recommend you move back to a previous version.

Reply:
@bsneed Ok thanks but unfortunatly I can't because we upgraded to the latest react native version and need to support iOS 10.
I will try to see if it works for us by patching this file back to iOS 10 (at least it builds, but I need to do regression testing).